conf/bookkeeper.conf

```diff
@@ -495,35 +480,34 @@ compactionRateByEntries=1000
 # Set the rate at which compaction will readd entries. The unit is bytes added per second.
 compactionRateByBytes=1000000
 
-#############################################################################
-## Statistics
-#############################################################################
-
-# Whether statistics are enabled
-# enableStatistics=true
-
-# Stats Provider Class (if statistics are enabled)
-statsProviderClass=org.apache.bookkeeper.stats.prometheus.PrometheusMetricsProvider
-
-# Default port for Prometheus metrics exporter
-prometheusStatsHttpPort=8000
+# Flag to enable/disable transactional compaction. If it is set to true, it will use transactional compaction,
+# which it will use new entry log files to store compacted entries during compaction; if it is set to false,
+# it will use normal compaction, which it shares same entry log file with normal add operations.
+# useTransactionalCompaction=false
```

Comment:
Since the python script to replace values in the file is only changing in uncommented lines, can we uncomment all the lines and leave the default values?

Reply:
done